packages/MarkdownEditor-Core.package/MarkdownToken.class/instance/extendWith..st

```diff
@@ -0,0 +1,4 @@
+extending
+extendWith: aCharacter
```

Comment:
Weird method name. I would like `append:` better.

I know you are trying to make it consistent with `isExtendableWith:` but I'd like to stick to convention and call it append.